src/core/constants.js

```diff
@@ -6,6 +6,13 @@
 
 const _PI = Math.PI;
 
+/**
+ * Version of this p5.js.
+ * @property {String} VERSION
+ * @final
+ */
+export const VERSION = '1.3.0';
```

Comment:
is there a way to link this to package.json so we can ensure this constant stays synced?

Reply:
It seems like it's possible with `require("./package.json")` so I will check this. Also I didn't notice that I opened a branch in processing's repository (I didn't know I have access) so I will close this for now and reopen on my user account's branch.